packages/web3-shared/src/hooks/useConstant.ts

```diff
@@ -2,6 +2,13 @@ import { useMemo } from 'react'
 import { ChainId, Web3Constants } from '../types'
 import { useChainId } from './useChainId'
 
+/**
+ * @deprecated Use useConstantNext from @dimensiondev/web3-shared package
+ *
+ * Before: `useConstant(T, "a")`
+ *
+ * After: `useConstantNext(T).a`
```

Comment:
```suggestion
 * Before: `useConstant(T, "a")`
 * After: `useConstantNext(T).a`
```

Reply:
![image](https://user-images.githubusercontent.com/5390719/122703559-72e41b80-d284-11eb-8e5a-5f460689268d.png)

The extra enter is required to make the JSDoc display correctly.